base/src/main/proto/spine/base/error.proto

```diff
@@ -50,10 +49,13 @@ message Error {
     // An integer code resulting from a native code call, or a number from a Proto enum value.
     int32 code = 2;
 
-    // An error indicating that a message did not pass validation.
-    validate.ValidationError validation_error = 3;
```

Comment:
Let's start by deprecating the field yet leaving it, so that we don't break any existing code. 

Please add the `[deprecated = true]` field option.

Reply:
This change was done to allow moving `ValidationError` into another submodule. If we make it just deprecated, we won't be able to move the type.